src/routes/projects-route.js

```diff
@@ -3,7 +3,9 @@ import express from 'express'
 import bibliographyRouter from './bibliography-route.js'
 import characterRouter from './characters-route.js'
 import documentRouter from './document-route.js'
+import duplicationRouter from './duplication-route.js'
```

Comment:
This does not exist in this PR and will cause the build to fail. Remove the reference to the non-existent files.